tests/test_state.py

```diff
@@ -24,14 +22,15 @@ def random_tensor(size=(4, 10)):
 def get_dummy_state():
     model = SimpleModel()
     optimizers = torch.optim.Adadelta(model.parameters())
-    state = State(model=model,
-                  run_name=f'{random.randint(0, 100)}',
-                  grad_accum=random.randint(0, 100),
-                  rank_zero_seed=random.randint(0, 100),
-                  precision=Precision.AMP,
-                  max_duration=f'{random.randint(0, 100)}ep',
-                  optimizers=optimizers,
-                  algorithms=[ChannelsLast()])
```

Comment:
why do we remove `ChannelsLast`?

Reply:
The `SimpleModel` has no convolution layers, so ChannelsLast has no effect